components/dropdown/dropdown.js

```diff
@@ -19,7 +19,7 @@ class Dropdown extends DropdownOpenerMixin(LitElement) {
 	getOpenerElement() {
 		return this.shadowRoot.querySelector('slot')
 			.assignedNodes()
-			.find(node => node.className === 'd2l-dropdown-opener');
+			.find(node => node.classList && node.classList.contains('d2l-dropdown-opener'));
```

Comment:
Nit: probably safe at this point given our supported browsers to assume the `classList` API exists

Reply:
It seems to be necessary because `assignedNodes` can return non-element nodes. For text nodes, `classList` is undefined.

Reply:
Ah gotcha 👍

Reply:
Does this flatten the tree when providing the nodes?  I think the original implementation did a query that may have worked regardless of where the opener was in the userland sub-tree, but I am not certain.

Reply:
`assignedNodes` does not flatten by default. I'll double check to see what the behavior is on the orginal.

Reply:
Looks like there is a `{ flatten: true }` option for the API if we need it.

Reply:
Double checked this; the original implementation only checks immediate children so we should be good without `flatten`.